backend/de.metas.business/src/main/sql/postgresql/ddl/functions/C_BPartner_Adv_Search_update.sql

```diff
@@ -1,23 +1,60 @@
 DROP FUNCTION IF EXISTS C_BPartner_Adv_Search_Update()
 ;
 
+CREATE FUNCTION c_bpartner_adv_search_update() RETURNS void
```

Comment:
imho this function is not needed

pls check `c_bpartner_adv_search_update(p_c_bpartner_ids numeric[])`,
calling it with p_c_bpartner_ids := null will do the same

Reply:
wdyt?

Reply:
hmm, now i saw that the function already existed, but you just inversed the order in this file.... so fine for me to keep it.

PS: when you perform such changes, before committing pls also do a review and see how the changes look like and if it will easy for the reviewer to understand them....